snare/tests/test_replace_links.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+import sys
+import os
+import shutil
+import asyncio
+from snare.cloner import Cloner
+from snare.utils.page_path_generator import generate_unique_path
+
+
+class TestReplaceLinks(unittest.TestCase):
+    def setUp(self):
+        self.main_page_path = generate_unique_path()
+        os.makedirs(self.main_page_path)
+        self.root = 'http://example.com'
+        self.level = 0
+        self.max_depth = sys.maxsize
+        self.loop = asyncio.new_event_loop()
+        self.css_validate = 'false'
+        self.handler = Cloner(self.root, self.max_depth, self.css_validate)
+        self.content = '''
```

Comment:
Make it None and specify its value where you are using it, do the same for some of the above files.